packages/@uppy/aws-s3/types/index.d.ts

```diff
@@ -10,8 +9,20 @@ export interface AwsS3UploadParameters {
     headers?: { [type: string]: string }
 }
 
-export interface AwsS3Options extends AwsS3MultipartOptions {
-    /** @deprecated future versions of this plugin will use the Expires value from the backend */
+export interface AwsS3Options {
+    id?: string
+    companionUrl?: string | null
+    companionHeaders?: Record<string, string>
+    allowedMetaFields?: Array<string> | null
+    getUploadParameters?: (file: UppyFile) => MaybePromise<{
+      method: "PUT" | "POST"
+      url: string
+      fields: Record<string, string> | null
+      headers: Record<string, string>
+    }>
+    limit?: number
+    getResponseData?: (responseText: string, response: XMLHttpRequest) => void
```

Comment:
Did not exist in existing type information. Should be correct based on docs(?)

Reply:
```suggestion
    /** @deprecated this method will not be supported in future versions of this plugin */
    getResponseData?: (responseText: string, response: XMLHttpRequest) => void
```